<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMCWidget</class>
 <widget class="QWidget" name="IMCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="IMCViewer" name="graphicsView"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QPushButton" name="pushButton_load">
       <property name="text">
        <string>加载图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addBox">
       <property name="text">
        <string>添加矩形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addCircle">
       <property name="text">
        <string>添加圆形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_color">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>画笔颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_compute">
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pan">
       <property name="text">
        <string>浏览模式</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_edit">
       <property name="text">
        <string>编辑模式</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pen">
       <property name="text">
        <string>画笔</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_eraser">
       <property name="text">
        <string>橡皮</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>画笔宽度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_brush">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
       <property name="sliderPosition">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_brush">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>橡皮宽度</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="horizontalSlider_eraser">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBox_eraser">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QSpinBox" name="spinBox_lhs">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRangeSlider" name="horizontalRangeSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_rhs">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>透明度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalOpySlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxOpy"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="text">
        <string>IMC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_name">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IMCViewer</class>
   <extends>QGraphicsView</extends>
   <header>IMCViewer.h</header>
  </customwidget>
  <customwidget>
   <class>CRangeSlider</class>
   <extends>QSlider</extends>
   <header>crangeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
